code-challenges.js

```diff
@@ -29,9 +51,46 @@ var nums1 = [3, 56, 90, -8, 0, 23, 6]
 var nums2 = [109, 5, 9, -59, 8, 24]
 // Expected output: [-59, 109]
 
+//function takes in array
+const minMax = (array) => {
+  //let newArray = []
+  //have to sort in order to put in order
+  let arraySorter = array.sort(function(a,b){return a-b})
+  //must filter to only include the vlaues at the extremes
+  let arrayFilter = arraySorter.filter((value, index) => {
+    return index === 0 || index == arraySorter.length - 1
+  })
+return arrayFilter
+  //return arraySorter
+}
+
+
+
+//wanted to take another crack at it using an empty array and pushing into it.
+const minMax2 = (array) => {
+  let finalArray = []
+  let arraySorter2 = array.sort(function(a,b){return a-b})
+  let popper = arraySorter2.pop()
+  let pusher = finalArray.push(popper)
+  let shifter = arraySorter2.shift()
+  let unshifter = finalArray.unshift(shifter)
+  return finalArray
```

Comment:
this is a fun solution too! we could even return a new array whose values are `[arraySorter2[0], arraySorter2[arraySorter.length - 1]]` just to bypass some of those lines if you wanted :)